star-dash/star-dash/GameEngine/Entities/EntityFactory.swift

```diff
@@ -46,6 +46,7 @@ struct EntityFactory {
                         size: size)
             .withPhysics(rectangleOf: PhysicsConstants.Dimensions.monster)
                 .configureCollisionBitMask(PhysicsConstants.CollisionMask.monster)
+                .configureIsDynamic(false)
```

Comment:
FYI. In my feat/monsters branch i set it as dynamic to allow it move about and collide. But you can leave this in for now.